### PR TITLE
Add scientist category to press page (Issue #159)

### PR DESCRIPTION
**Pull request by jfoucry**
Add ## Scientist publications in French; Scientist publication in English; Publications scientifique francophone ; Publications scientifique anglophone in pages `press.fr.md` and `press.en.md`, with *INRIA* publication about video games on smartphone.

### CODE REVIEW

File: content/page/press.en.md [hunk 1]
```diff
@@ -100,3 +102,6 @@ draft: false
 * **LCI** - *Nov 29 2017* - [Ce qu'il faut savoir sur les trackers présents dans nos applications mobiles](https://www.lci.fr/high-tech/une-invasion-de-trackers-dans-nos-applications-le-bon-coin-allo-cine-mobiles-android-iphone-2071872.html)
 * **Next Inpact** - *Nov 25 2017* - [Rencontre avec Exodus Privacy, qui révèle les trackers](https://www.nextinpact.com/news/105655-rencontre-avec-exodus-privacy-qui-revele-trackers-applications-android.htm)
 * **Le Monde** - *Nov 24 2017* - [Des mouchards cachés dans vos applications pour smartphones](http://www.lemonde.fr/pixels/article/2017/11/24/des-mouchards-caches-dans-vos-applications-pour-smartphones_5219892_4408996.html)
+## Scientist publications in French
```
Comment (DavidLibeau) [inline content/page/press.en.md:105]: Je mettrais plus "Scientific publication" (c.f. https://www.sciencedirect.com/topics/social-sciences/scientific-publications)

Comment (jfoucry) [reply content/page/press.en.md:105]: C'est ce que je voulais mettre mais je me suis basé sur le titre de l'issue.

File: content/page/press.en.md [hunk 2]
```diff
@@ -98,5 +102,11 @@ draft: false
 * **Numerama** - *Dec 8 2017* - [Lutter contre les mouchards des apps, une cause citoyenne](https://www.numerama.com/politique/313309-lutter-contre-les-mouchards-des-apps-une-cause-citoyenne-voici-lhistoire-dexodus-privacy.html)
 * **Tom's Guide** - *Dec 5 2017* - [Trois quarts des apps Android espionnent les utilisateurs](https://www.tomsguide.fr/actualite/applications-android-espionnage-utilisateurs,60333.html)
 * **LCI** - *Nov 29 2017* - [Ce qu'il faut savoir sur les trackers présents dans nos applications mobiles](https://www.lci.fr/high-tech/une-invasion-de-trackers-dans-nos-applications-le-bon-coin-allo-cine-mobiles-android-iphone-2071872.html)
+
```
Comment (pnu-s) [inline content/page/press.en.md:105]:
```suggestion
```

File: content/page/press.en.md [hunk 3]
```diff
@@ -98,5 +102,11 @@ draft: false
...
 * **Next Inpact** - *Nov 25 2017* - [Rencontre avec Exodus Privacy, qui révèle les trackers](https://www.nextinpact.com/news/105655-rencontre-avec-exodus-privacy-qui-revele-trackers-applications-android.htm)
+
```
Comment (pnu-s) [inline content/page/press.en.md:107]:
```suggestion
```

File: content/page/press.en.md [hunk 4]
```diff
@@ -47,6 +47,10 @@ draft: false
 * **BoingBoing** - *Nov 25 2017* - [Researchers craft Android app that reveals to find horrific menagerie](https://boingboing.net/2017/11/25/la-la-la-cant-hear-you.html)
 * **The Intercept** - *Nov 24 2017* - [Staggering Variety of Clandestine Trackers Found in Popular](https://theintercept.com/2017/11/24/staggering-variety-of-clandestine-trackers-found-in-popular-android-apps/)
 
+## Scientifict publications in English
+
+* **INRIA** — *Apr 25 2022* — [The price to play](https://dl.acm.org/doi/abs/10.1145/3485447.3512279)
```
Comment (pnu-s) [inline content/page/press.en.md:52]:
```suggestion
* **INRIA** - *Apr 25 2022* - [The price to play](https://dl.acm.org/doi/abs/10.1145/3485447.3512279)
```

File: content/page/press.en.md [hunk 5]
```diff
@@ -98,5 +102,11 @@ draft: false
...
 * **Tom's Guide** - *Dec 5 2017* - [Trois quarts des apps Android espionnent les utilisateurs](https://www.tomsguide.fr/actualite/applications-android-espionnage-utilisateurs,60333.html)
 * **LCI** - *Nov 29 2017* - [Ce qu'il faut savoir sur les trackers présents dans nos applications mobiles](https://www.lci.fr/high-tech/une-invasion-de-trackers-dans-nos-applications-le-bon-coin-allo-cine-mobiles-android-iphone-2071872.html)
+
 * **Next Inpact** - *Nov 25 2017* - [Rencontre avec Exodus Privacy, qui révèle les trackers](https://www.nextinpact.com/news/105655-rencontre-avec-exodus-privacy-qui-revele-trackers-applications-android.htm)
+
 * **Le Monde** - *Nov 24 2017* - [Des mouchards cachés dans vos applications pour smartphones](http://www.lemonde.fr/pixels/article/2017/11/24/des-mouchards-caches-dans-vos-applications-pour-smartphones_5219892_4408996.html)
+
+## Scientifict publications in French
+
+* **INRIA** — *Apr 25 2022* — [Jeux vidéo sur mobile : protéger la vie pvivée en ligne](https://www.inria.fr/fr/jeux-video-smartphones-securite-donnees-personnelles-internet)
```
Comment (pnu-s) [inline content/page/press.en.md:112]:
```suggestion
* **INRIA** - *Apr 25 2022* - [Jeux vidéo sur mobile : protéger la vie pvivée en ligne](https://www.inria.fr/fr/jeux-video-smartphones-securite-donnees-personnelles-internet)
```

File: content/page/press.fr.md [hunk 6]
```diff
@@ -63,6 +63,10 @@ draft: false
 * **Next Inpact** - *25 Nov 2017* - [Rencontre avec Exodus Privacy, qui révèle les trackers](https://www.nextinpact.com/news/105655-rencontre-avec-exodus-privacy-qui-revele-trackers-applications-android.htm)
 * **Le Monde** - *24 Nov 2017* - [Des mouchards cachés dans vos applications pour smartphones](http://www.lemonde.fr/pixels/article/2017/11/24/des-mouchards-caches-dans-vos-applications-pour-smartphones_5219892_4408996.html)
 
+## Parutions Scientifiques Francophone
```
Comment (pnu-s) [inline content/page/press.fr.md:66]:
```suggestion
## Parutions scientifiques francophones
```

File: content/page/press.fr.md [hunk 7]
```diff
@@ -63,6 +63,10 @@ draft: false
...
+
+* **INRIA** — *25 avr 2022* — [Jeux vidéo sur mobile : protéger la vie privée en ligne](https://www.inria.fr/fr/jeux-video-smartphones-securite-donnees-personnelles-internet)
```
Comment (pnu-s) [inline content/page/press.fr.md:68]:
```suggestion
* **INRIA** - *25 Avr 2022* - [Jeux vidéo sur mobile : protéger la vie privée en ligne](https://www.inria.fr/fr/jeux-video-smartphones-securite-donnees-personnelles-internet)
```

File: content/page/press.fr.md [hunk 8]
```diff
@@ -105,3 +109,7 @@ draft: false
 * **Gizmodo** - *28 Nov 2017* - [Study: Vast Majority of Google Play Apps Are Covertly Tracking Users](https://gizmodo.com/study-vast-majority-of-google-play-apps-are-covertly-t-1820821682)
 * **BoingBoing** - *25 Nov 2017* - [Researchers craft Android app that reveals to find horrific menagerie](https://boingboing.net/2017/11/25/la-la-la-cant-hear-you.html)
 * **The Intercept** - *24 Nov 2017* - [Staggering Variety of Clandestine Trackers Found in Popular](https://theintercept.com/2017/11/24/staggering-variety-of-clandestine-trackers-found-in-popular-android-apps/)
+
+## Parutions Scientifiques anglophone
```
Comment (pnu-s) [inline content/page/press.fr.md:113]:
```suggestion
## Parutions scientifiques anglophones
```

File: content/page/press.fr.md [hunk 9]
```diff
@@ -105,3 +109,7 @@ draft: false
...
+
+* **INRIA** – *25 apr 2022* – [The price to play](https://dl.acm.org/doi/abs/10.1145/3485447.3512279)
```
Comment (pnu-s) [inline content/page/press.fr.md:115]:
```suggestion
* **INRIA** - *25 Avr 2022* - [The price to play](https://dl.acm.org/doi/abs/10.1145/3485447.3512279)
```

**Review by pnu-s — CHANGES_REQUESTED** [review]
Thanks for the PR!

Made a couple of suggestions

File: content/page/press.en.md (hunk 3 above)
Comment (jfoucry) [reply content/page/press.en.md:107]: Those are missing?

**Review by pnu-s — APPROVED** [review]

**Comment by pnu-s** [comment]
Fixes #159

**Pull request merged**